nextcord/voice_recording/errors.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: MIT
+
+import nextcord.errors as nc_errors
```

Comment:
why?

Reply:
import as prevents circular imports because it waits for the module to be ready before continuing